desktop/src/util.rs

```diff
@@ -118,22 +130,17 @@ pub fn winit_to_ruffle_key_code(event: &KeyEvent) -> Option<KeyCode> {
         Key::Character("x") => KeyCode::X,
         Key::Character("y") => KeyCode::Y,
         Key::Character("z") => KeyCode::Z,
-        Key::Character(";") => KeyCode::Semicolon,
-        Key::Character("=") => KeyCode::Equals,
-        Key::Character(",") => KeyCode::Comma,
-        Key::Character("-") if event.location == KeyLocation::Numpad => KeyCode::NumpadMinus,
-        Key::Character("-") => KeyCode::Minus,
-        Key::Character(".") if event.location == KeyLocation::Numpad => KeyCode::NumpadPeriod,
-        Key::Character(".") => KeyCode::Period,
-        Key::Character("/") if event.location == KeyLocation::Numpad => KeyCode::NumpadSlash,
-        Key::Character("/") => KeyCode::Slash,
-        Key::Character("`") => KeyCode::Grave,
-        Key::Character("[") => KeyCode::LBracket,
-        Key::Character("\\") => KeyCode::Backslash,
-        Key::Character("]") => KeyCode::RBracket,
-        Key::Character("'") => KeyCode::Apostrophe,
-        Key::Character("*") => KeyCode::Multiply,
-        Key::Character("+") => KeyCode::Plus,
+        Key::Character(";") | Key::Character(":") => KeyCode::Semicolon,
```

Comment:
Does this make sense? We are mapping logical key codes to something totally different.

Reply:
My main concern is that this is very much tied to a QWERTY keyboard layout. What happens with a different keyboard layout in Flash Player?

Reply:
This does not make sense at all, but does FP make sense?

1. (English US) When pressing <kbd>Shift</kbd> + <kbd>3</kbd>, you get codes 16 (for Shift) and 51 (for 3).
2. (English UK) When pressing <kbd>Shift</kbd> + <kbd>3</kbd>, you get codes 16 (for Shift) and 163 (for £, because there is <kbd>£</kbd> on UK).
3. (English US) When pressing <kbd>Backslash</kbd>, you get 220 (for backslash).
4. (English UK) When pressing <kbd>Backslash</kbd>, you get 51 (for key 3, because there is <kbd>#</kbd> on UK).
5. (English US) When pressing <kbd>Shift</kbd> + <kbd>Backslash</kbd>, you get 16 (for Shift) and 220 (for backslash).
6. (English UK) When pressing <kbd>Shift</kbd> + <kbd>Backslash</kbd>, you get 16 (for Shift) and 192 (for `, because there is <kbd>~</kbd> on UK).

Reply:
I just tested with a German keyboard layout, and I can reproduce that `:` and `;`, which are totally different keys on the German keyboard, produce 186. :exploding_head: